frontend/src/tests/setup.js

```diff
@@ -66,6 +66,18 @@ class DatabaseSeeder {
                     email: "megan.miller@utoronto.ca",
                     utorid: "millerm",
                 },
+                {
+                    last_name: "Lucas",
+                    first_name: "George",
+                    email: "george.lucas@utoronto.ca",
+                    utorid: "lucasg",
+                },
+                {
+                    last_name: "Bellford",
```

Comment:
Maybe `Bell`? We don't want to be accidentally associated with Jordan Belfort...